mllib/src/test/scala/org/apache/spark/ml/feature/RFormulaSuite.scala

```diff
@@ -313,13 +306,14 @@ class RFormulaSuite extends MLTest with DefaultReadWriteTest {
       Seq(("male", "foo", 4), ("female", "bar", 4), ("female", "bar", 5), ("male", "baz", 5))
         .toDF("id", "a", "b")
     val model = formula.fit(original)
+    val attr = NominalAttribute.defaultAttr
     val expected = Seq(
         ("male", "foo", 4, Vectors.dense(0.0, 1.0, 4.0), 1.0),
         ("female", "bar", 4, Vectors.dense(1.0, 0.0, 4.0), 0.0),
         ("female", "bar", 5, Vectors.dense(1.0, 0.0, 5.0), 0.0),
         ("male", "baz", 5, Vectors.dense(0.0, 0.0, 5.0), 1.0)
     ).toDF("id", "a", "b", "features", "label")
-    // assert(result.schema.toString == resultSchema.toString)
+      .select($"id", $"a", $"b", $"features", $"label".as("label", attr.toMetadata()))
```

Comment:
nit: indent

Reply:
It was at the level of val +2 extra spaces. Should I indent the dots to the same row?

Reply:
I think maybe
```
...
).toDF("id", "a", "b", "features", "label")
 .select($"id", ...
```
looks beautiful.

Reply:
I am sorry to spending time with this issue but I would like to be consistent and keep the rules so what about the following:

~~~
...
)
  .toDF("id", "a", "b", "features", "label")
  .select($"id", ...
~~~

So all indented by two spaces and the dots are aligned. Could you accept this?

Reply:
I am also confused about the align rule. @jkbradley what do you think ?